### PR TITLE
fix:  height style issue

### PR DESCRIPTION
**Pull request by ohhoney1**
fix the style issue when the number of slides is less than or equal to slidesToShow.

https://github.com/ant-design/ant-design/issues/44766

### CODE REVIEW

File: src/inner-slider.js [hunk 1]
```diff
@@ -751,6 +751,7 @@ export class InnerSlider extends React.Component {
 
     if (this.props.unslick) {
       listProps = { className: "slick-list" };
+      trackProps.trackStyle = null;
```
Comment (afc163) [inline src/inner-slider.js]: We should change this line: https://github.com/ant-design/react-slick/blob/3343a0d00fb5bf16edee1a78608e6702be17aae3/src/utils/innerSliderUtils.js#L594

to 

```js
  const trackChildren = spec.unslick ? spec.slideCount : spec.slideCount + 2 * spec.slidesToShow;
```

Comment (ohhoney1) [reply src/inner-slider.js]: ok

File: src/track.js [hunk 2]
```diff
@@ -135,7 +135,7 @@ const renderSlides = (spec) => {
     );
 
     // if slide needs to be precloned or postcloned
-    if (spec.infinite && spec.fade === false) {
+    if (spec.infinite && spec.fade === false && !spec.unslick) {
```
Comment (ohhoney1) [inline src/track.js:138]: VerticalMode.js 例子中，Slider 子元素数量改为 1 ，由于设置了infinite: true，导致出现了克隆元素

**Comment by afc163** [comment]
能补一下用例不

**Comment by ohhoney1** [comment]
> 能补一下用例不

done.

**Pull request merged**